CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
> Two ticks are easier to type ;)

I don't know which keyboard you are but if I type once the backtick touch, I directly have a backtick.

Anyway, we should not use it "because it's easy for some people".

Stop me if I'm wrong, but one-line code with markdown is write with only one backtick, and 3 for multi-lines.

It would be great to follow these specs. :+1: